gprofiler/profilers/java.py

```diff
@@ -5,12 +5,15 @@
 import errno
 import functools
 import os
+import re
 import shutil
+from dataclasses import dataclass
```

Comment:
Does it work with Python 3.6? (the standalone executable uses it)

Reply:
No it doesn't, I'll remove it